common/dtd/ldml.dtd

```diff
@@ -2126,6 +2126,15 @@ $Revision$
 <!ATTLIST minusSign numberSystem CDATA #IMPLIED >
     <!--@DEPRECATED-->
 
+<!ELEMENT approximatelySign ( #PCDATA ) >
+<!ATTLIST approximatelySign alt NMTOKENS #IMPLIED >
+    <!--@MATCH:literal/variant-->
+<!ATTLIST approximatelySign draft (approved | contributed | provisional | unconfirmed | true | false) #IMPLIED >
```

Comment:
remove " | true | false"

(no need to add born-deprecated items)

Reply:
Done